packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -565,16 +558,36 @@ export default {
     }
 }
 
-.c-cookieBanner-cta {
+.c-cookieBanner-cta-container {
     padding-block: f.spacing(d) f.spacing(a);
     padding-inline: f.spacing(e);
 
+    .c-cookieBanner-cta {
+        width: 100%;
+    }
+
+    .c-cookieBanner-cta:not(:last-child) {
+        margin-bottom: f.spacing(d);
+    }
+
     @include f.media('>=mid') {
+        display: flex;
+        flex-direction: row-reverse;
         padding-block: f.spacing(d);
         padding-inline: f.spacing(d);
+        margin-inline: f.spacing(d);
         min-width: 320px;
```

Comment:
This is one I missed originally, this should be min-inline-size

Reply:
fixed.